tests/fixtures.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable max-len */
```

Comment:
If you use tick mark (`) characters, eslint should not complain.

Reply:
I just tried with tick marks and it still complained, are you okay with merging it as is?

Reply:
Yeah. Definitely no need to block on this one.